### PR TITLE
i227: Refactor form validators to turnery operators

### LINKED ISSUE #227: Refactor form validators to use turnery operators
#### problem

When refactoring form validators to use turnery operators instead of if else statement, form validation breaks. 

On submit, form field validators aren't being called. 

#### current code
```Dart
validator: (value) {
          if (value.isEmpty) {
            return 'please enter password again';
          } else if (!validRepeatPassword(value, password)) {
            return 'passwords do not match';
          } else {
            return null;
          }
```

#### code wished to be implemented

```Dart
validator: (value) => value.isEmpty
            ? 'please enter password again'
            : (!validPassword(value))
                ? 'passwords do not match'
                : null,
```

### PR DESCRIPTION
**Pull request by nickmeinhold**
Fixes #227

Seems to be all good - tests all pass and manual testing shows validators working as expected

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -872,5 +872,5 @@ packages:
     source: hosted
```
Comment (nickmeinhold) [inline pubspec.lock:872]: caused by latest 'dev' channel update

Comment (tamari-gray) [reply pubspec.lock:872]: sweet

**Review by tamari-gray — APPROVED** [review]
looks good!
happy you got that working!

**Comment by nickmeinhold** [comment]
thanks mate!

**Pull request merged**